scripts/ci/fd-conformance.mjs

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env zx
+
+// Firedancer conformance testing of the Core BPF Config program against its
+// original builtin implementation.
+//
+// Note: This script can only be run on Ubuntu.
+
+import 'zx/globals';
+import { getProgramId, getProgramSharedObjectPath, workingDirectory } from '../utils.mjs';
+
+// Clone the conformance harness.
+const harnessPath = path.join(workingDirectory, 'solana-conformance');
+await $`git clone https://github.com/firedancer-io/solana-conformance.git`;
+
+// Clone the test vectors.
+const testVectorsPath = path.join(harnessPath, 'impl', 'test-vectors');
+await $`git clone https://github.com/firedancer-io/test-vectors.git ${testVectorsPath}`;
+
+// Add the Mollusk-generated fixtures to the test inputs.
+const firedancerFixturesPath = path.join(testVectorsPath, 'instr', 'fixtures', 'config');
```

Comment:
Looking at the test-vectors repo at https://github.com/firedancer-io/test-vectors/tree/main/instr/fixtures, this probably makes more sense, right?
```suggestion
const firedancerFixturesPath = path.join(testVectorsPath, 'instr', 'fixtures', 'address-lookup-table');
```

Reply:
Haha whoops! I'm not sure how that got changed, maybe bad copy-paste.